simulariumio/tests/converters/test_cytosim_converter.py

```diff
@@ -23,7 +23,7 @@
                     "fibers": CytosimObjectInfo(
                         filepath="simulariumio/tests/data/cytosim"
                         "/3_fibers_3_frames/fiber_points.txt",
-                        agents={0: CytosimAgentInfo(name="fiber")},
+                        agents={0: CytosimAgentInfo(name="fiber", radius=1e-3)},
```

Comment:
Why was this change necessary?

Reply:
because radii are multiplied by scale factor. I thought they were before this PR but anyway this makes sense and is needed for the test to pass